platforms/core-configuration/object-graph-serialization/src/main/kotlin/org/gradle/configurationcache/serialization/Combinators.kt

```diff
@@ -275,104 +219,34 @@ suspend fun <K, V, T : MutableMap<K, V>> ReadContext.readMapEntriesInto(items: T
 }
 
 
-internal
```

Comment:
Extension functions moved to platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/configurationcache/serialization/ClassPathEncodingExtensions.kt